include/realizations/catchment/Formulation_Manager.hpp

```diff
@@ -9,7 +9,11 @@
 #include <functional>
 #include <dirent.h>
 #include <sys/stat.h>
+#include <errno.h>
```

Comment:
```suggestion
#include <cerrno>
```

nitpick: `<errno.h>` is for the C standard library, `<cerrno>` is the C++ counterpart.

Reply:
Missed this one.